scripts/check-docs-quality.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const { execSync, spawnSync } = require('child_process');
```

Comment:
Just a thought. Can `vale` be used as a library? Might be easier to just use the library programmatically rather then shelling out to just call the value CLI.

Reply:
Unfortunately not. Only method is to use it as cli (https://docs.errata.ai/vale/install). Moreover cli doesn't provide any flags utility to ignore directories.